frontend/src/metabase/parameters/utils/dashboards.unit.spec.js

```diff
@@ -191,65 +190,6 @@ describe("metabase/parameters/utils/dashboards", () => {
     });
   });
 
-  describe("getParametersMappedToDashcard", () => {
```

Comment:
Tests moved to `frontend/src/metabase/dashboard/actions/getNewCardUrl.unit.spec.ts`